packages/neos-ui/src/index.js

```diff
@@ -75,20 +88,10 @@ async function main() {
     reloadNodes();
 }
 
-function initializeFrontendConfiguration() {
```

Comment:
Sadly this breaks 4 of my plugins, as I use `globalRegistry.get('frontendConfiguration')` via the `neos` decorator.

And I'm quite sure, that other plugins use this to.

Reply:
ah sorry, I access the editPreviewModes and allowedTargetWorkspaces in the commandbar plugin. The frontendConfiguration still seems to work.

Reply:
hi yes `globalRegistry.get('frontendConfiguration')` should still continue to work.

The new location of editPreviewModes and allowedTargetWorkspaces is intentional one is now available via `getConfiguration()` and the other now resides in the redux store:

https://github.com/neos/neos-ui/pull/4012#issuecomment-3466687435

how did you access the neos configuration for `allowedTargetWorkspaces`?

I can brew a commit which makes the `editPreviewModes` and `allowedTargetWorkspaces` available as beforehand but at least with 9.1 id like to remove this legacy layer then.